### PR TITLE
Fix bug in updating video stream

### PR DESCRIPTION
**Pull request by GodVenn**
The video streams were not being properly updated.
Refactored and also included what was updated in the logs.

### CODE REVIEW

**Comment by GodVenn** [comment]
> 🔔 Changes in database folder detected 🔔 Do these changes require **adding new migrations**? 🤔 In that case follow [these steps](https://github.com/equinor/flotilla/tree/main/backend#Database-model-and-EF-Core). If you are uncertain, ask a database admin on the team 😄

Not needed

File: backend/api/Database/Models/VideoStream.cs [hunk 1]
```diff
@@ -25,5 +25,45 @@ public class VideoStream
         public string Type { get; set; }
 
         public bool ShouldRotate270Clockwise { get; set; }
+
+        public bool Equals(VideoStream other)
```
Comment (eivindsjovold) [inline backend/api/Database/Models/VideoStream.cs:29]: other? non-descriptive name

**Review by eivindsjovold — COMMENTED** [review]
Any tests to this functionality?

**Review by Christdej — APPROVED** [review]
LGTM

File: backend/api/Database/Models/VideoStream.cs (hunk 1 above)
Comment (GodVenn) [reply backend/api/Database/Models/VideoStream.cs:29]: The class of the other describes it

**Pull request merged**